bin/jsonwebtokenydoo.js

```diff
@@ -186,6 +192,7 @@ async function main() {
     )} ✨`
     // here is a request you can make to get you started
   );
+  clipboardy.writeSync(data.token);
```

Comment:
We probably want to add some output saying that the token has been copied to the clipboard?